.github/workflows/lint.yml

```diff
@@ -0,0 +1,37 @@
+name: Lint
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: ruby/setup-ruby@v1
```

Comment:
Did this not specify the with `bundler-cache: true` option for a reason?

Reply:
The decision to ignore the project's Gemfile in our linting process is intentional.

This eliminates the need to keep our project Gemfile synchronised with updates to Brakeman. This approach aims to speed up the linting process.